meson_options.txt

```diff
@@ -15,7 +15,7 @@ option('openblas_include',
 
 option('opencl_include',
        type: 'array',
-       value: ['/usr/include/'],
+#       value: ['/usr/include/'],
```

Comment:
I had to comment this path as it seems to confuse the Android NDK, it mixes the headers and fail compilation. The GNU toolchain doesn't have that problem.